pkg/kubelet/apis/cri/services.go

```diff
@@ -43,6 +43,8 @@ type ContainerManager interface {
 	ListContainers(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error)
 	// ContainerStatus returns the status of the container.
 	ContainerStatus(containerID string) (*runtimeapi.ContainerStatus, error)
+	// UpdateContainerResources updates the cgroup resources for the container.
```

Comment:
yes, since those are the resources supported in the docker engine API.  The supported resources are here https://github.com/kubernetes/kubernetes/pull/46105/files#diff-c7dd39479fd733354254e70845075db5R408